### PR TITLE
show pod IP during debug

### PR DESCRIPTION
**Pull request by deads2k**
This came up while I was fighting with nginx.  I needed to know the pod IP so that I could curl it.  This isn't very useful for remote debugging sessions, but debugging pods when you're running in the infrastructure or when you can start up a busybox pod in the same namespace, this is useful.

@stevekuznetsov @fabianofranz

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
> This isn't very useful for remote debugging sessions

Aren't most deployments going to be like this for customers? You can get this information if you need it using `jsonpath` or similar, right? I'm not sure we need this.

**Comment by deads2k** [comment]
> Aren't most deployments going to be like this for customers? You can get this information if you need it using jsonpath or similar, right? I'm not sure we need this.

You can't actually, not in the same window.  Once you drop to `oc debug` you're stuck and can't get back out.  A remote debugging session could use a busybox image running in the same namespace to hit the IP that they get out of this command.

**Comment by stevekuznetsov** [comment]
That's a solid argument to expose the IP during debug to make that workflow reasonable. Do we have an issue to track the proxy so that this could be useful for all clusters?

**Comment by deads2k** [comment]
> That's a solid argument to expose the IP during debug to make that workflow reasonable. Do we have an issue to track the proxy so that this could be useful for all clusters?

Yeah, clayton's got a followup issue for `oc debug`.

File: pkg/cmd/cli/cmd/debug.go [hunk 1]
```diff
@@ -444,6 +444,12 @@ func (o *DebugOptions) Debug() error {
 		case err != nil:
 			return err
 		default:
+			// TODO this doesn't do us much good for remote debugging sessions, but until we get a local port
+			// set up to proxy, this is what we've got.
+			if podWithStatus, ok := containerRunningEvent.Object.(*kapi.Pod); ok {
+				fmt.Fprintf(o.Attach.Err, "Pod IP: %s\n", podWithStatus.Status.PodIP)
```
Comment (stevekuznetsov) [inline pkg/cmd/cli/cmd/debug.go]: Why print to the error output?

Comment (deads2k) [inline pkg/cmd/cli/cmd/debug.go]: > Why print to the error output?

It's where the rest are going: https://github.com/openshift/origin/pull/8724/files#diff-79c5f976ae3273934daeada9309b517dR396

**Comment by stevekuznetsov** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

File: pkg/cmd/cli/cmd/debug.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/debug.go]: Any informational messages here go to stderr.

**Pull request merged**